dev/tests/verification/Tests/ExecuteInSeleniumTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace tests\verification\Tests;
+
+use tests\util\MftfTestCase;
+
+class ExecuteInSeleniumTest extends MftfTestCase
+{
+    /**
+     * Tests generation of executeInSelenium action.
+     *
+     * @throws \Exception
+     * @throws \Magento\FunctionalTestingFramework\Exceptions\TestReferenceException
+     */
+    public function testExecuteJsTest()
```

Comment:
Possible renaming this to testExecuteInSeleniumTest()?

Reply:
Yes of course

Reply:
@jilu1 I have changed test name.
Please proceed with review